packages/material-ui/src/styles/experimentalStyled.d.ts

```diff
@@ -180,9 +180,8 @@ export interface StyledOptions {
 }
 
 interface MuiStyledOptions<Theme extends object = any> {
-  displayName?: string;
-  className?: string;
-  muiName?: string;
+  name: string;
+  slot: string;
```

Comment:
Both should be optional?

Reply:
Good catch, yes they should be optional.

Reply:
One thing I was thinking about. If no `slot` is specified, but the `name` is, should we consider it is `Root`?

Reply:
If this is simpler for developers using styled(), why not

Reply:
Done